changelog/unreleased/pull-272

```diff
@@ -0,0 +1,12 @@
+Enhancement: can now listen on a unix socket
```

Comment:
Please change the title to `Enhancement: Support listening on a unix socket`

Reply:
Modified as suggested.